bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/MappingSpec.java

```diff
@@ -352,4 +358,104 @@ public String toString() {
 		builder.append("]");
 		return builder.toString();
 	}
+
+	/**
+	 * @return <code>true</code> if the RIF layout has an inner join relationship
+	 *         <code>false</code> if not
+	 */
+	public Boolean getHasInnerJoinRelationship() {
+		return this.hasInnerJoinRelationship;
+	}
+
+	/**
+	 * @param hasInnerJoinRelationship
+	 *            the new value for {@link #getHasInnerJoinRelationship()}
+	 * @return this {@link MappingSpec} instance, for call-chaining purposes
+	 */
+	public MappingSpec setHasInnerJoinRelationship(Boolean hasInnerJoinRelationship) {
+		this.hasInnerJoinRelationship = hasInnerJoinRelationship;
+		return this;
+	}
+
+	/**
+	 * @param innerJoinRelationship
+	 *            a list of {@link String} parameters defining the inner join
+	 *            relationship for the entity
+	 * @return this {@link MappingSpec} instance, for call-chaining purposes
+	 */
+	public MappingSpec setInnerJoinRelationship(List<List<String>> innerJoinRelationship) {
```

Comment:
Why'd you decide to go with the loosely typed `List<List<String>>` approach here instead of a more struct-like-class approach such as `List<InnerJoinRelationship>`?